packages/rich-text/src/plugins/Hyperlink/index.tsx

```diff
@@ -101,7 +104,10 @@ function EntityHyperlink(props: HyperlinkElementProps) {
   const isReadOnly = useReadOnly();
   const sdk: FieldExtensionSDK = useSdkContext();
   const { target } = props.element.data;
-
+  const { onEntityFetchComplete } = props;
+  React.useEffect(() => {
+    onEntityFetchComplete?.();
```

Comment:
Maybe leave a comment here that we do this immediately as we lazy load the entity and therefore consider this ready with the first render.